.pre-commit-config.yaml

```diff
@@ -5,6 +5,7 @@ repos:
   - id: black
     types_or: [python]
     args: ["--config=./pyproject.toml"]
+    exclude: "hadamard_utils.py"
```

Comment:
Why do we need to exclude that file?

Reply:
It's mostly a file full of tensors of +/-1s, and black takes ages to run through it (minutes), and does style really make sense when it's a tensor?